src/dimension_vector.c

```diff
@@ -51,11 +63,11 @@ ts_dimension_vec_create(int32 initial_num_slices)
 }
 
 DimensionVec *
-ts_dimension_vec_sort(DimensionVec **vecptr)
+ts_dimension_vec_sort(DimensionVec **vecptr, bool reverse)
```

Comment:
Could you use a separate function `ts_dimension_vec_sort_reverse` or even something like `enum SortDirection { FORWARD, REVERSE }` for this? Jumping into one of the calling functions and just seeing `ts_dimension_vec_sort(vec, false)` isn't the most obvious thinng

Reply:
done